osu.Framework/Graphics/UserInterface/TabControl.cs

```diff
@@ -246,23 +287,16 @@ protected virtual void AddTabItem(TabItem<T> tab, bool addToDropdown = true)
         /// Removes a <see cref="TabItem{T}"/> from this <see cref="TabControl{T}"/>.
         /// </summary>
         /// <remarks>
-        /// <para>
-        /// If <paramref name="tab"/> is currently selected and is not the only tab in the <see cref="TabControl{T}"/>,
-        /// then selection will switch to the next or previous one depending on <paramref name="tab"/>'s position.
-        /// </para>
-        /// <para>
-        /// If <paramref name="tab"/> is currently selected and is the only tab in the <see cref="TabControl{T}"/>,
-        /// then selection will be cleared.
-        /// </para>
+        /// If the current tab is removed and <see cref="SwitchTabOnRemove"/> is <c>true</c>, then selection will change to a new tab if possible or be cleared if there are no tabs remaining in the <see cref="TabControl{T}"/>.
         /// </remarks>
         /// <param name="tab">The tab to remove.</param>
-        /// <param name="removeFromDropdown">Whether the tab should be removed from the Dropdown if supported by the <see cref="TabControl{T}"/> implementation.</param>
+        /// <param name="removeFromDropdown">Whether the tab should be removed from the dropdown if supported by the <see cref="TabControl{T}"/> implementation.</param>
```

Comment:
gonna add the suggestion here too to speed things up, if you think it's a good idea

```suggestion
        /// <param name="removeFromDropdown">Whether the tab should be removed from the <see cref="Dropdown"/> if supported by the <see cref="TabControl{T}"/> implementation.</param>
```